.github/pull_request_template.md

```diff
@@ -27,4 +27,17 @@ close a related issues using keywords (https://help.github.com/articles/closing-
 @mentions (https://help.github.com/articles/basic-writing-and-formatting-syntax/#mentioning-people-and-teams)
 of the person or team responsible for reviewing proposed changes. -->
 
+## PR Checklist
+- [ ] [Makes breaking changes](https://raw.githubusercontent.com/InsightSoftwareConsortium/ITK/master/CONTRIBUTING.md#making-breaking-changes)
+- [ ] [Makes design changes](https://raw.githubusercontent.com/InsightSoftwareConsortium/ITK/master/CONTRIBUTING.md#making-design-changes)
```

Comment:
*making-design-changes* ->*design-changes*

Reply:
:eyes: Done in 453f1e9. Thanks !